lib/pyspark_preprocesing.py

```diff
@@ -0,0 +1,17 @@
+from pyspark.ml.feature import StringIndexer, OneHotEncoder
+from pyspark.sql import DataFrame
+from typing import List
+
+
+def one_hot_encoding(df: DataFrame, columns: List[str]) -> DataFrame:
+    out_cols = [x + '_ohe' for x in columns]
+    out_num = [x + '_num' for x in columns]
+
+    df_num = StringIndexer(inputCols=columns,
```

Comment:
Brakuje mi obsługi wyjątków, w naszym przypadku, nulli. Ponieważ nie wiemy ile będzie jeszcze odfiltrowane z braków, to posłużyłbym się funkcją setHandleInvalid(). Przykładowo:
```
def one_hot_encoding(df: DataFrame, columns: List[str], exception = "skip") -> DataFrame:
...
df_num = StringIndexer(inputCols=columns,
                           outputCols=out_num).setHandleInvalid(exception).fit(df).transform(df)
...
```

Reply:
setHandleInvalid dodane